solution02/src/test/resources/user.properties

```diff
@@ -47,8 +47,8 @@ Fusername=testuserFacebook
 Fpassword=facebook123
 role=roleFacebook
 permission=/permission/admin
-FbClientId=214847805723276
-FbSecret=47d454e7798a92ce71588148b90497eb
+FbClientId=368299723596958
+FbSecret=a627a9fbee0832a4576e4d19be4a0e6b
```

Comment:
IMO we should not have the client id and secret here. This should be blank and needs to documented.